events/views.py

```diff
@@ -198,14 +198,27 @@ class MeetingViewSet(viewsets.ModelViewSet):
 
 
 def get_next_meeting():
-    return Meeting.objects\
-                  .filter(event__start__gte=datetime.now())\
-                  .order_by('event__start')[0]
+    return (
+        Meeting.objects
+        .filter(
+            event__start__gte=datetime.now(),
+            ongoing=False
+        )
+        .order_by('event__start')
+        .first()
+    )
 
 
 @private_api(point=str)
 def add_point_to_next_meeting(request, point):
     meeting = get_next_meeting()
+    if meeting is None:
+        return JsonResponse(
```

Comment:
If I understand correctly, the `private_api` protocol would be to return 
```json
{"error": "...",
 "hint": "..."}
```
for all errors ?